litmus-portal/frontend/src/views/WorkflowDetails/WorkflowStatus/index.tsx

```diff
@@ -0,0 +1,42 @@
+import { Typography } from '@material-ui/core';
+import React from 'react';
+import useStyles from './styles';
+
+interface WorkflowStatusProps {
+  phase: string;
+}
+
+const WorkflowStatus: React.FC<WorkflowStatusProps> = ({ phase }) => {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.status}>
+      <span className={classes.icon}>
+        <img
+          className={
+            phase.toLowerCase() === 'running' ? classes.runningSmallIcon : ''
+          }
+          src={`/icons/${phase.toLowerCase()}.svg`}
+          alt="status"
+        />
+      </span>
+      <Typography>
+        <span
+          className={`${
+            phase === 'Succeeded'
+              ? classes.succeeded
```

Comment:
you can refactor this logic into a function that takes the phase and gives a className back using a switch-case...It'll be a bit cleaner to read.